app/ui/src/app/customizations/api-connector/api-connector-list.component.html

```diff
@@ -44,8 +46,8 @@ <h2>API Client Connector</h2>
                 </div>
               </div>
               <div class="list-pf-additional-content">
-                <div *ngIf="item.connectors">
-                  <strong>Used by {{ item.connectors }}</strong>
+                <div *ngIf="item.apiConnectors">
+                  <strong>Used by {{ item.apiConnectors }}</strong>
```

Comment:
May be you'll want to implement a 118nPluralPipe here to refine the text output, like `Used by 1 connector`, `Used by 23 connectors`, `Used by no connectors yet`... 

Here you have a [link to the API specification](https://angular.io/api/common/I18nPluralPipe) and also an actual [example on Syndesis](https://github.com/syndesisio/syndesis/pull/556/files#diff-8cb4ab233e8d35edb6d1bd79beaeb809R19).

Reply:
Definitely, but since we don't even have the values yet I didn't write out the complete statement (which should end with 'connection' or 'connections' obviously). I can add it in a separate PR once that's ready.

Reply:
Why wait? Having no data is a good way to also test how the app behaves in edge cases! Go ahead and do some nice experiments with text here. Since `item.apiConnectors` will yield nothing, you can see how a `no connections` or `0 connections` would fit in the overall UI.

;)